bin/merged_prs.sc

```diff
@@ -144,11 +144,13 @@ def template(
       |For full details: https://github.com/scalameta/metals/milestone/<num>?closed=1
       |
       |Metals is a language server for Scala that works with VS Code, Vim, Emacs and
-      |Sublime Text. Metals is developed at the
-      |[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+      |Sublime Text. Metals is developed at [VirtusLab](https://virtuslab.com)
       |with the help from [Lunatech](https://lunatech.com) along with contributors from
       |the community.
       |
+      |Metals was created at the [Scala Center](https://scala.epfl.ch) and is being
```

Comment:
@julienrf I changed the default message as we discussed, let me know if this is ok.

Reply:
The change looks fine to me but I think we should first make a public announcement on the Scala blog. I am swamped until the 18th, so I propose postponing the changes to the next release in case you want to publish the release notes for 0.11.9 soon.

Reply:
> @julienrf I changed the default message as we discussed

Could you provide a bit more context here. Is this line meant to replace the one above it?

Reply:
I will postpone the change for now.